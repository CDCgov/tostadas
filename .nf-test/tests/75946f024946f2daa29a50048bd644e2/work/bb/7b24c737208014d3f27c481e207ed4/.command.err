WARNING: Skipping mount /var/singularity/mnt/session/etc/resolv.conf [files]: /etc/resolv.conf doesn't exist in container
