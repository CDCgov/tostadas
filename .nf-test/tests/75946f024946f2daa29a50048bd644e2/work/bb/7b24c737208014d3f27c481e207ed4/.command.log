WARNING: Skipping mount /var/singularity/mnt/session/etc/resolv.conf [files]: /etc/resolv.conf doesn't exist in container
refgff is ref.MPXV.NC063383.v7.gff
0 374
ID=ITR-UPDATETAG-RPT1;rpt_type=long_terminal_repeat ID=ITR-UPDATETAG-RPT2;rpt_type=long_terminal_repeat
