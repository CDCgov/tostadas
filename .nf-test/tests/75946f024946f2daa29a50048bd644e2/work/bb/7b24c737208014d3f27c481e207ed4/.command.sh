#!/bin/bash -ue
repeatmasker_liftoff.py         --repeatm_gff NY0006.fasta.out.gff         --liftoff_gff NY0006.liftoff-orig.gff         --refgff ref.MPXV.NC063383.v7.gff         --fasta NY0006.fasta          --sample_name NY0006
