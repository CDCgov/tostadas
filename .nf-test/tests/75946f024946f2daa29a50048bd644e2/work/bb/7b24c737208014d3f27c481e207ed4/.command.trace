nextflow.trace/v2
realtime=3000
%cpu=1522
cpu_model=Intel(R) Xeon(R) Gold 6154 CPU @ 3.00GHz
rchar=18456452
wchar=135995
syscr=1828
syscw=17
read_bytes=21236736
write_bytes=147456
%mem=0
vmem=5010852
rss=72388
peak_vmem=5017016
peak_rss=73848
vol_ctxt=4743
inv_ctxt=21313
