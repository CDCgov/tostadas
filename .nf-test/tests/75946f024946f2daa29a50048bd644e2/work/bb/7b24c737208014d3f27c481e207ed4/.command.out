refgff is ref.MPXV.NC063383.v7.gff
0 374
ID=ITR-UPDATETAG-RPT1;rpt_type=long_terminal_repeat ID=ITR-UPDATETAG-RPT2;rpt_type=long_terminal_repeat
